common.py

```diff
@@ -8,22 +9,24 @@
                                      'v2.0.x', 'v3.0.x'],
                  'astropy/astropy-helpers': ['v0.4.x', 'v1.0.x', 'v1.1.x',
                                              'v1.2.x', 'v1.3.x',
-                                             'v2.0.x', 'v3.0.x']}
+                                             'v2.0.x', 'v3.0.x'],
+                 'astropy/astroquery': []  # we don't have bugfix branches
+                 }
```

Comment:
How about making this dictionary a defaultdict that just returns an empty dictionary?

Reply:
Actually looking at this again, I think I would prefer keep it as is we may not want to avoid handling not knowing about the branches in https://github.com/astropy/astropy-procedures/pull/99/files#diff-cf5a98d3b8ded947cd1a470ce7e3769eR53. This way there is a way to distinguish cases where we know there are no branches from the ones where there may be branches we just don't know about them.